types/nationalAvalancheCenter/observations.ts

```diff
@@ -242,3 +246,127 @@ export type PartnerType = (typeof PartnerType)[keyof typeof PartnerType];
 export const FormatPartnerType = (value: PartnerType): string => {
   return reverseLookup(PartnerType, value);
 };
+// from the client-side schema at https://github.com/NationalAvalancheCenter/nac-vue-component-library/blob/main/constants/observations.js
+export const observationSchema = z.object({
+  organization: z.string().nullable(),
+  center_id: z.preprocess(s => String(s).toUpperCase(), avalancheCenterIDSchema),
+  observer_type: z.nativeEnum(PartnerType),
+  name: z.string().nullable(),
+  phone: z.string().nullable(),
+  email: z.string().email().nullable(),
+  created_at: z.string().nullable(),
+  start_date: z.string().nullable(),
+  end_date: z.string().nullable(),
+  activity: z.array(z.nativeEnum(Activity)),
+  location_point: z
+    .object({
+      lng: z.number().nullable(),
+      lat: z.number().nullable(),
+    })
+    .nullable(),
+  location_name: z.string(),
+  route: z.string(),
+  instability: z.object({
+    avalanches_observed: z.boolean(),
+    avalanches_triggered: z.boolean(),
+    avalanches_caught: z.boolean(),
+    cracking: z.boolean(),
+    cracking_description: z.nativeEnum(InstabilityDistribution).or(z.string().length(0)).nullable(),
+    collapsing: z.boolean(),
+    collapsing_description: z.nativeEnum(InstabilityDistribution).or(z.string().length(0)).nullable(),
+  }),
+  instability_summary: z.string(),
+  observation_summary: z.string(),
+  media: z.array(mediaItemSchema).nullable(),
+  // urls: z.array(z.string()).nullable(), // what even is this?
```

Comment:
I think this corresponds to uploading video URLs from YouTube/Insta/etc

Reply:
Ah, crap ... would have thought those were in media but not image types ... sigh